### PR TITLE
Seori/DSODA-204

### PR DESCRIPTION
**Pull request by msbeethoven**
Last fixes: Hover states for social and sticky header overlap fix

### CODE REVIEW

File: app/components/show-header/template.hbs [hunk 1]
```diff
@@ -1,19 +1,27 @@
 <div class="static-main-header">
-  <div class="col" id="wqxr-logo" data-test-element="header-logo">{{nypr-a-svg className="site-header-title-logo" icon="wqxr-log-frq"}}</div>
+  <div class="col" id="wqxr-logo" data-test-element="header-logo">
+    {{nypr-a-svg className="site-header-title-logo" icon="wqxr-log-frq"}}</div>
```
Comment (bkammin) [inline app/components/show-header/template.hbs]: you can clean this up a bit by moving the `</div>` to a new line at the same indentation level as its corresponding `<div>`

File: app/styles/app.scss [hunk 2]
```diff
@@ -36,13 +36,4 @@ body.ember-application{
...
   @include btn;
-}
-
-a:hover {
-	opacity: 80%;
-}
-
-.c-toggle-box__label {
-  // @todo 	remove when we replace the toggle box component
-  font-family: $font-body;
```
Comment (bkammin) [inline app/styles/app.scss]: I added this code recently -- let's keep it!

File: app/templates/application.hbs [hunk 3]
```diff
@@ -23,14 +27,8 @@
...
-    @composerName={{this.currentStream.composerName}}
-    @streamSlug={{this.currentStream.slug}}
-    @streamName={{this.currentStream.name}}
-  />
-{{/animated-if}}
+<JukeboxPlayer @trackTitle={{this.currentStream.trackTitle}} @showTitle={{this.currentStream.showTitle}}
+  @episodeTitle={{this.currentStream.episodeTitle}} @ensembleName={{this.currentStream.ensembleName}}
+  @conductorName={{this.currentStream.conductorName}} @composerName={{this.currentStream.composerName}}
+  @streamSlug={{this.currentStream.slug}} @streamName={{this.currentStream.name}} />
+{{/animated-if}}
```
Comment (bkammin) [inline app/templates/application.hbs]: looks like this formatting got messed up, let's keep it clean!

File: app/styles/app.scss (hunk 2 above)
Comment (msbeethoven) [reply app/styles/app.scss]: Sorry, deleted the wrong thing in a merge conflict. Added back.

**Pull request merged**